defaults/main.yml

```diff
@@ -6,9 +6,13 @@ logcheck_install:
 logcheck_reportlevel: server
 logcheck_sendmailto: root
 logcheck_syslogsummary: true
+logcheck_sortuniq: false
 logcheck_logfiles:
   - /var/log/syslog
   - /var/log/auth.log
   - /var/log/mail.log
   - /var/log/kern.log
 logcheck_custom_ignores: []
```

Comment:
I agree, but then we have to tag an new major release. And I prefer defaults that match our use-case :-)